.github/workflows/kafka_demo_test.yml

```diff
@@ -53,9 +53,9 @@ jobs:
           echo "Extracting Environment Variables"
           echo "Triggering event is ${{ github.event_name }}"
           if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-            echo "key-release-image=mcr.microsoft.com/aci/skr:2.7" >> $GITHUB_OUTPUT
-            echo "consumer-image=mcr.microsoft.com/acc/samples/kafka/consumer:2.0" >> $GITHUB_OUTPUT
-            echo "producer-image=mcr.microsoft.com/acc/samples/kafka/producer:2.0" >> $GITHUB_OUTPUT
+            echo "key-release-image=${{ vars.RELEASED_SKR_IMAGE }}" >> $GITHUB_OUTPUT
```

Comment:
since these images are public, I don't think they need to be secrets...

Reply:
They are in env vars, not in secrets.